src/h3/_cy/cells.pyx

```diff
@@ -305,53 +301,51 @@ cpdef double mean_hex_area(int resolution, unit='km^2') except -1:
     try:
         area *= convert[unit]
     except:
-        raise H3ValueError('Unknown unit: {}'.format(unit))
+        raise ValueError('Unknown unit: {}'.format(unit))
 
     return area
 
 
 cpdef double cell_area(H3int h, unit='km^2') except -1:
     cdef:
-        h3lib.H3Error err
         double area
 
-    check_cell(h)
-
     if unit == 'rads^2':
         err = h3lib.cellAreaRads2(h, &area)
     elif unit == 'km^2':
         err = h3lib.cellAreaKm2(h, &area)
     elif unit == 'm^2':
         err = h3lib.cellAreaM2(h, &area)
     else:
-        raise H3ValueError('Unknown unit: {}'.format(unit))
+        raise ValueError('Unknown unit: {}'.format(unit))
+
+    check_for_error(err)
 
     return area
 
 
+cdef couldnt_find_line(err, start, end):
```

Comment:
It might be best to avoid contractions in function names

Reply:
Agreed.